provision/PROVISION.md

```diff
@@ -0,0 +1,53 @@
+## Provision
+
+>Note: This provisioning model only supports Keycloak Legacy distribution, we are working on a version to support Quarkus based distribution.
+
+#### Keycloak
+A light weight Keycloak server setup with, PostgreSQL DB as its external store using docker-compose.
+
+ - The image used for Keycloak server is the latest:legacy container from [quay.io](https://quay.io/repository/keycloak/keycloak?tab=tags&tag=legacy)
+ - To control the cpu quota for the containers, we use cpusets for both containers (keycloak and postgres), that can be changed from the [keycloak-legacy-postgres.yml](keycloak/keycloak-legacy-postgres.yml)
+ 
+##### Pre-requisite
+The keycloak-legacy-postgres.yml file should work with the below docker and docker-compose versions
```

Comment:
Be brave and remove "should" and "probably" from this file in all places. This makes the user wonder when this wouldn't be the case.